### PR TITLE
Add `Steps` component to `add-view-transitions.mdx`

### PR DESCRIPTION
**Pull request by jacobdalamb**
#### Description (required)

Add Starlight's `Steps` component to `add-view-transitions.mdx`

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

File: src/content/docs/en/tutorials/add-view-transitions.mdx [hunk 1]
```diff
@@ -53,6 +54,7 @@ Read more about [Astro's view transitions](/en/guides/view-transitions/) in our
 <Box icon="question-mark">
 ### Test your knowledge
 
+<Steps>
```
Comment (sarah11918) [inline src/content/docs/en/tutorials/add-view-transitions.mdx:57]:
```suggestion
```

File: src/content/docs/en/tutorials/add-view-transitions.mdx [hunk 2]
```diff
@@ -94,14 +96,15 @@ Read more about [Astro's view transitions](/en/guides/view-transitions/) in our
       </Option>
     </MultipleChoice>
 
+</Steps>
```
Comment (sarah11918) [inline src/content/docs/en/tutorials/add-view-transitions.mdx:99]:
```suggestion
```

**Review by sarah11918 — COMMENTED** [review]
Thanks @jacobdalamb ! The multiple choice questions aren't "steps" though, so I've removed them. Otherwise, this looks good! I'm enjoying our new Step component!

**Review by liruifengv — APPROVED** [review]
Looks Great! LGTM! Thanks @jacobdalamb

**Pull request merged**